cwl/workflow/data_assembly_DNA_wf.cwl

```diff
@@ -54,7 +62,7 @@ steps:
     in:
       input_SV: input_SV
       conditional_run: gatekeeper/scatter_WGS
-  #  scatter: conditional_run
+    scatter: conditional_run
```

Comment:
Like I mentioned in the tool, you should look into implementing cwl 1.2's conditional workflows. It will make this bit less confusing.